infra/Pos/Util/TimeLimit.hs

```diff
@@ -60,7 +61,10 @@ logWarningLongAction secure delta actionTag action =
     -- 'withAsync' is assumed to take care of this, and indeed it does for
     -- 'Production's implementation, which uses the definition from the async
     -- package: 'uninterruptibleCancel' is used to kill the thread.
-    withAsync (waitAndWarn delta) (const action)
+    --
+    -- thinking even more about it, unmasking auxilary thread is crucial if
+    -- this function is going to be called under 'mask'.
```

Comment:
Shouldn't it be OK so long as it's not called in an *uninterruptible* masking state?

Where are the call sites in which `logWarningLongAction` is called in masked state?

Reply:
https://issues.serokell.io/issue/CSL-1837

`applyBlocks` calls `onApplyBlocks` from `BListener` which uses `logWarningLongAction` in wallet's implementation (`onApplyBlocksWebWallet`).

Reply:
Actually, I wanted it to work under `MaskedUninterruptibly` state only because in future there accidentally could be a similar case, when someone calls `bracket` or smth other big piece of code, and later someone another inserts innocent `logWarningWait`...